pkg/certificateprovider/certificate_provider.go

```diff
@@ -25,10 +25,13 @@ import (
 // This is a map containing Cert store configuration including name, tenantID, and cert object information
 type CertStoreConfig map[string]string
 
+// This is a map of properties for fetched certificates
```

Comment:
nit: probably we could add comment on the key and value meaning as well.

Reply:
thanks Binbin! i have added comments on the method